guides/v2.1/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -9,40 +9,40 @@ redirect_from:
   - /guides/v2.1/extension-dev-guide/backward-compatibility.html	
 ---
 
-Every Magento version release include a change in the codebase.
+Every Magento version release includes a change in the codebase.
 The scope of the change determines whether the MAJOR, MINOR, or PATCH number increases in the version.
 
 ## Public vs private code changes
 
 A Magento module's codebase consists of public and private code.
 Public code includes Public [API](https://glossary.magento.com/api)(indicated by the `@api` docblock tag) and Public Customization Points.
-Private code is not accessible via the API: classes and constants marked as [private][].
+All other code is considered private.
 
 Changes in public code always trigger MINOR or MAJOR version increases.
 
 In most cases, modifications to private code will trigger PATCH version increases.
 On rare occasions, if the Magento development team made significant modifications to private code, they will increase the MINOR or MAJOR version.
 
-Third-party modules should not use or change private code.
-If this is unavoidable, you must depend on the patch version of the core modules used.
+Third-party modules should not rely on private code.
+If this is unavoidable, you must depend on the PATCH version of the core modules used.
```

Comment:
We should discuss what this means if we start using four-segment/hotfix version numbers

Reply:
If we will, all documents that mentioning versioning should be reviewed and updated. Agree.